lib/shopify_app/sessions_concern.rb

```diff
@@ -87,5 +88,16 @@ def return_address
       session.delete(:return_to) || main_app.root_url
     end
 
+    def perform_after_authenticate_job
+      config = ShopifyApp.configuration.after_authenticate_job
+
+      return unless config && config[:job].present? && config[:inline].in?([true, false])
+
+      if config[:inline]
```

Comment:
Yeah, `config[:inline] == true` is good because it's self documenting code. If someone thinking about running an inline job, reading this code would tell them that they need to set that value to `true`.